assembly/runtime/runtime.ts

```diff
@@ -17,19 +18,33 @@ export type HeaderType = Header;
 export type BlockType = Block;
 export type Moment = UInt64;
 export type NonceType = UInt64;
+export type AmountType = UInt64;
 export type BlockHashCount = UInt32;
 export type ExtrinsicIndex = UInt32;
-export type InherentType = Inherent;
-export type SignedTransactionType = SignedTransaction;
+export type SignedTransactionType = SignedTransaction<HashType, AmountType, NonceType, SignatureType>;
+export type ISignedTransactionType = ISignedTransaction<HashType, AmountType, NonceType, SignatureType>;
```

Comment:
We should think about whether we need both `SignedTransactionType` and `ISignedTransactionType` in the runtime ts config. Is one of them redundant? We can discuss it.